contribution/contributions.md

```diff
@@ -146,8 +146,13 @@ We are following Google Code style:
 * [Java style](https://google.github.io/styleguide/javaguide.html)
 * [Shell style](https://google.github.io/styleguide/shell.xml)
 
-Check style report location are in `${submodule}/target/site/checkstyle.html`
-Test coverage report location are in `${submodule}/target/site/cobertura/index.html`
+There are some plugins to format, lint your code in IDE (use [_tools/checkstyle.xml](https://github.com/apache/zeppelin/tree/master/_tools) as rules)
+
+* [Checkstyle plugin for Intellij](http://stackoverflow.com/questions/26955766/intellij-idea-checkstyle)
```

Comment:
<img width="1121" alt="screen shot 2016-11-29 at 12 56 30 pm" src="https://cloud.githubusercontent.com/assets/4968473/20695976/4fa3c3d6-b633-11e6-859d-70ff11236362.png">

I thought people would search `checkstyle` in their IDEA preference like the image above.
Let me replace the link with https://plugins.jetbrains.com/plugin/1065